apps/testingaccessibility/src/pages/api/stripe/checkout.ts

```diff
@@ -181,6 +178,10 @@ export default withSentry(async function stripeCheckoutHandler(
         )
       }
 
+      const successUrl = isUpgrade
+        ? `${process.env.NEXT_PUBLIC_URL}/welcome?session_id={CHECKOUT_SESSION_ID}?upgrade=true`
+        : `${process.env.NEXT_PUBLIC_URL}/thanks/purchase?session_id={CHECKOUT_SESSION_ID}`
```

Comment:
with an upgrade we know they are logged in because you can't upgrade so let's send them to welcome and not the thanks "check your inbox" page. The email still gets sent and they can click it to also get to welcome page.

below if they happen to hit welcome and aren't logged in, it forwards them to thanks as a fallback